<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>698</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>885</width>
    <height>698</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>885</width>
    <height>698</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Админка</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="vopros">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>881</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить вопрос</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Лучший результат во всём мире:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="luchadm">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить 4 варианта ответов:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btndob">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="admerror">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btndel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить базу вопросов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ссюда ввести правильный вариант ответа</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dob1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>207</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dob2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dob3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>207</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dob4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="togame">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>610</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>В игровое меню</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>581</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="butdelone">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>470</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>удалить выбранный вопрос</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
